hack/stabilization-changes.py

```diff
@@ -33,10 +33,11 @@ def parse_iso8601_delay(delay):
     # https://tools.ietf.org/html/rfc3339#page-13
     match = _ISO_8601_DELAY_REGEXP.match(delay)
     if not match:
-        raise ValueError('invalid or unsupported ISO 8601 duration {!r}.  Tooling currently only supports P<number>WT<number>H for week and/or hour offsets')
+        raise ValueError('invalid or unsupported ISO 8601 duration {!r}.  Tooling currently only supports P<number>W for weeks, or P<number>DT<number>H for day/hour offsets'.format(delay))
```

Comment:
I wonder if we have any python library to do the same thing rather than doing it ourself.

Reply:
I could not find one back when I initially implemented this script, but I'm not against slotting one in if someone else knows of one.

Reply:
Two more projects - [arrow](https://github.com/arrow-py/arrow) and [pyiso8610](https://github.com/micktwomey/pyiso8601) - are parsing timestamps, but not durations. Perhaps we should pick one and contribute parsing code there?

Reply:
Agree, its probably not worth it at this point